e2e-tests/cypress/tests/integration/channels/channel/convert_group_message_to_private_spec.js

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channels @channel
+
+describe('Group Message', () => {
```

Comment:
Wonder if you want to add some case (or just make sure the situation is the handles that) for the issue we found today about the URL not showing the correct target team.

Reply:
I've not fixed it yet. Will address it in https://mattermost.atlassian.net/browse/MM-54724 .

Reply:
Handled in unit tests.